site/GrapherWithFallback.tsx

```diff
@@ -0,0 +1,36 @@
+import { Grapher } from "@ourworldindata/grapher"
+import React from "react"
+import { GrapherFigureView } from "./GrapherFigureView.js"
+import { BAKED_GRAPHER_EXPORTS_BASE_URL } from "../settings/clientSettings.js"
+import cx from "classnames"
+
+export const GrapherWithFallback = ({
+    grapher,
+    slug,
+    className,
+}: {
+    grapher?: Grapher | undefined
+    slug?: string
+    className?: string
+}) => {
+    return slug ? (
+        <div className={cx("GrapherWithFallback", className)}>
+            <>
+                {grapher ? (
+                    <GrapherFigureView grapher={grapher} />
+                ) : (
+                    // Given GrapherFigureView also renders a figure which gets
+                    // the same styling and height, rendering an empty <figure>
+                    // while Grapher is instanciated probably helps with
```

Comment:
instantiated*

this is just to have an element to be styled while Grapher loads in, right?